### DISCUSSION TITLE
How to Change Casks Installation Path and Keep Old Version of Casks?

### DISCUSSION THREAD

**Discussion by BingoKingo** [Homebrew/discussions#3883] [Getting started] [answered]
Hi, I am a newbie. I add `export HOMEBREW_NO_INSTALL_CLEANUP=1` and `export HOMEBREW_CASK_OPTS='--appdir=/Applications/Casks --fontdir=~/Library/Fonts'` in `~/.zshrc` but only work in zsh, not work for sh, bash and other shells. 
So, how to make it available for other shells?

**Comment by SMillerDev** [comment, marked as answer]
https://stackoverflow.com/questions/6372751/what-is-the-difference-between-the-various-shell-profiles has information about shell profile files

**Reply by BingoKingo** [reply]
I'd like to know where the following variables stored in by default, so that I can put them into together?
```
PATH='/opt/homebrew/bin:/opt/homebrew/sbin'
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
MANPATH=/opt/homebrew/share/man::
```
